code/game/machinery/bots/secbot.dm

```diff
@@ -349,6 +353,13 @@ Auto Patrol: []"},
 			mode = BOT_HUNT
 			spawn(0)
 				bot_process()	// ensure bot quickly responds to a perp
+			if(fast == 0)
+				fast = 1
+				spawn(fast_length * 10)
```

Comment:
I have tested this on a local copy, and it works correctly. What's more is that if you attack Beepsky he will chase you as usual but not use this ability, it's only when he finds a player whose set to arrest